ui/package.json

```diff
@@ -8,6 +8,7 @@
     "patternfly": "^3.52.2",
     "patternfly-react": "^2.11.1",
     "react": "^16.4.1",
+    "react-copy-to-clipboard": "^5.0.1",
```

Comment:
This should allow patch and minor updates [1], which should not introduce any breaking change.

[1] https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004

Reply:
@jhellar following some parts of the doc added by you. 
 
`The `ˆ` : Allows changes that do not modify the left-most non-zero digit in the [major, minor, patch]`

`Many authors treat a 0.x version as if the x were the major "breaking-change" indicator.`

Because of it IHMO the best approach is to use `~` instead of `^`, besides by concept the break changes according to SemVer be Just in the X.0.0 some authors may add it in 0.X.0.